dist/api_err.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         result of a system crash. The application may choose to salvage the
         file or retry wiredtiger_open with the 'salvage=true' configuration
         setting.'''),
+    Error('WT_NONE', -32000,
```

Comment:
Maybe `WT_OK`?

Reply:
The basic default code that the ticket wants to have is WT_NONE, although I've just realised that the code should be 0 instead of -32000.

Reply:
Never mind, that is untrue, the `WT_NONE` should still be treated as an error, so -3200 is fine.

Reply:
Yea that does make sense. I'll add it to the Jira for discussion, just because I want to stick to the tech design for now. If others agree, then I'll make the change.

Reply:
I suggest we stick with `WT_NONE`, at least for now, as that is what is in the Tech Design. This code is intended to indicate there there is no sub-level error code for a particular error code, so 'none' makes sense. If there is an actual error, but an error that has no sub-level error code, then `WT_OK` could be confusing.

Reply:
I’d just prefer the identifier to be more informative. When I see `WT_NONE`, it’s unclear what exactly is "not there". Perhaps something like `WT_NO_ERROR` would be more descriptive.
I understand you’re short on time, so if it's not feasible to make it better now, we can do it later. There aren’t any external users for this yet anyway.
I’m not necessarily requesting this to be addressed in this PR.

Reply:
Thanks @ershov , something along those lines (perhaps WT_NO_SUB_ERROR) is worth considering (after this PR)